Assets/MRTK/Providers/OpenXR/Scripts/OpenXRHandJointProvider.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+#if MSFT_OPENXR && (UNITY_STANDALONE_WIN || UNITY_WSA)
+using Microsoft.MixedReality.OpenXR;
+#else
+using UnityEngine.XR;
+#endif // MSFT_OPENXR && (UNITY_STANDALONE_WIN || UNITY_WSA)
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.OpenXR
+{
+    internal class OpenXRHandJointProvider
```

Comment:
Feels weird to call this an OpenXRHandJointProvider since it also handles getting handjoint data in non-OpenXR pipelines. Would it be alright to more simply call it the HandJointProvider?

Reply:
Oh, good point that this is more generally usable (partially at least, for the non-`#MSFT_OPENXR` portions)! Since it's `internal`, I may merge it as-is and think about the relationship between it and a more general XR SDK hand joint provider for a future rework, which could be used by the Windows XR Plugin implementation as well. Will give this a thought before deciding to merge or not

Reply:
Okay, haven't had time to jump back to this yet, so I'm going to merge this and ensure the Windows XR Plugin part is accounted for later in this work